spring-kafka-test/src/main/java/org/springframework/kafka/test/context/EmbeddedKafka.java

```diff
@@ -87,8 +88,11 @@
 	int partitions() default 2;
 
 	/**
-	 * Topics that should be created
-	 * Topics may contain property placeholders, e.g. {@code topics = "${kafka.topic.one:topicOne}"}
+	 * Topics that should be created Topics may contain property placeholders, e.g.
+	 * {@code topics = "${kafka.topic.one:topicOne}"} The topics will be created with
+	 * {@link #partitions()} partitions; to provision other topics with other partition
+	 * counts call the {@code addPartitions(NewTopic... topics)} method on the autowired
```

Comment:
Must be `addTopics()`.
Fixing one merge...

Reply:
thx